src/libpriv/rpmostree-core.c

```diff
@@ -1336,19 +1336,19 @@ find_pkg_in_ostree (OstreeRepo     *repo,
         return FALSE;
 
       if (!same_pkg_chksum)
-        goto done; /* Note early happy return */
+        return TRUE; /* Note early return */
     }
 
-  in_ostree = TRUE;
+  /* We found an import, let's load the sepolicy state */
+  *out_in_ostree = TRUE;
+  gboolean selinux_match = FALSE;
```

Comment:
Nit: since we're unconditionally stomping on the output vars right from the start, let's just pass `out_selinux_match` directly to `commit_has_matching_sepolicy` ?